proxy/src/main/java/com/velocitypowered/proxy/connection/backend/LoginSessionHandler.java

```diff
@@ -65,6 +71,26 @@ public class LoginSessionHandler implements MinecraftSessionHandler {
     this.resultFuture = resultFuture;
   }
 
+  @Override
+  public void activated() {
+    if (serverConn.getPlayer().getConnection().getType() == ConnectionTypes.MODERN_FORGE) {
```

Comment:
It would be helpful to explain _why_ we need this logic. I know why (the client is renegotiating the handshake with backend mods, and if someone goes wrong on the server side then there is no way to recover) but this needs to be documented.

Reply:
I believe what I've written is accurate but probably worth double checking

Reply:
Aside from some typos that explanation looks good.